src/components/tooltip/Tooltip.tsx

```diff
@@ -254,7 +258,10 @@ class Tooltip extends React.Component<TooltipProps, State> {
         }
 
         if (showTooltip) {
-            componentProps['aria-describedby'] = this.tooltipID;
+            if (!isLabelMatchingTooltipText || childAriaLabel === undefined) {
```

Comment:
Do we have any unit tests for this change?

Reply:
yes `when aria-label does not exists describedBy relation should exists` 
https://github.com/box/box-ui-elements/blob/30f5e4d5d533be5a7de20804cc705d4ae818c5b8/src/components/tooltip/__tests__/Tooltip.test.tsx#L122

Reply:
It looks like that's testing the positive case. Do we have any assertions for the negative case?

Reply:
https://github.com/box/box-ui-elements/pull/2544/commits/31c7a92b5882470b4fd730fd35ffd6f8f5a28b36